# Msg for command output from NMPC

Header header

float64[] uOpt
